agent/consul/acl_client.go

```diff
@@ -130,3 +117,8 @@ func (c *Client) ResolveTokenAndDefaultMeta(token string, entMeta *structs.Enter
 
 	return authz, err
 }
+
+func (c *Client) updateSerfTags(key, value string) {
```

Comment:
Curious why you decided to pull this into a function? To match the `Server.updateSerfTags` function?

Reply:
Consistency mostly with a secondary goal of preventing future mistakes where  we update the serf tags for one serf pool but not others. The clients currently only have a single pool though so this isn't entirely necessary.